tests/system/feature_store/test_feature_store.py

```diff
@@ -601,8 +606,13 @@ def test_ingest_partitioned_by_key_and_time(
 
         file_system = fsspec.filesystem("v3io")
         kind = TargetTypes.parquet
-        path = f"{get_default_prefix_for_target(kind)}/sets/{name}-latest"
-        path = path.format(name=name, kind=kind, project=self.project_name)
+        path = f"{get_default_prefix_for_target(kind)}/sets/{name}"
+        path = path.format(
+            name=name,
+            kind=kind,
+            project=self.project_name,
+            run_uuid=measurements.status.targets[0].run_uuid,
+        )
```

Comment:
Given my comments above - it may be an interesting test to add where you change the default target path config, and also the place-holder config parameter, and verify that the code still works... As noted, I expect it to explode at the moment.

Reply:
like mentioned before: those things shouldn't be changed if we want to leave it backward compatible.
E.g: feature set got targets on its status, the target containing a path including {run_uuid} - the new code will fail when trying to replace the new tag.